### ISSUE TITLE
"Break Inner Beam" command actually joins Beams in certain combination of rhythms

### ISSUE THREAD

**Issue by Skip12ish** [musescore/MuseScore#25280]
### Issue type

UX/Interaction bug (incorrect behaviour)

### Description with steps to reproduce

1. Note  entry mode (time signature 2/4)
2. Enter 8th note, 16th rest, 16th note (beat 1 of measure)
3. Enter four 16th notes (beat 2 of measure)
4. Observe that beat 1 group is not beamed
5. To Beam beat 1 group:
6. Select notes and rest in beat 1
7. Use "Break Inner Beams" (Properties-Notation-Beam)
8. Beat 1 group is now beamed
Is this a bug?

### Supporting files, videos and screenshots

[MuseScore Lesson from Dorico.pdf](https://github.com/user-attachments/files/17480919/MuseScore.Lesson.from.Dorico.pdf)

### What is the latest version of MuseScore Studio where this issue is present?

4.4.2.242570931

### Regression

I was unable to check

### Operating system

Windows 11, Mac OS 10.14.6 (Mojave)

### Additional context

I sent the Musescore file as a pdf. If you start a new Score and enter the first few bars or more and follow my description above, you'll hopefully see what I mean. BTW-the pdf is only 12 bars of solo violin. 

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by MarcSabatella** [comment]
This is correct behavior.  The beam property you are seeing it about breaking all beams *except* the eighth note beam, and that's exactly what it does.  The naming is perhaps less than ideal, though.  And of course, it would be possible to totally redesign how all of this works.  but anyhow, currently, this is the way to create 8th note sub-beams.  To see the intended effect, try selecting the third of the group of four on the second beat.  It will break the second beam but leave the first:

![image](https://github.com/user-attachments/assets/8b1f972a-ac63-448b-8190-4b3e7aee6b11)

BTW, to join the beam on the first beat, don't select the notes - select only the rest.  Then click join beams.  Selecting the notes could also cause the first note of the group to be joined into the previous measure, which is presumably not what you want.

**Comment by Skip12ish** [comment]
Well thanks so much for your response and explanations! I'm coming from finale and your videos have been tremendous! I'm almost fully converted to Musescore and you've been a big part of that!
So for my issue, it wasn't intuitive, but I observed that it was consistent. Using the Break Inner Beam seemed to be the only way to join those those. I saw on one of  your videos about selecting the rest and tried that, but it didn't seem to work. I probably wasn't doing it right. I'll try that again.
As you've surmised, it's not intuitive because, in this case, there are no Beams initially, so it's not obvious that Beams need to be broken. So for me this is what I call an acceptable quirk and finale had plenty of them. So all is good!
Big Thanks Marc!!!

**Comment by MarcSabatella** [comment]
No, sorry, I think you're still misunderstanding  You're right that using a break beam command to join beams would be counterintuitive - that's why you're not supposed to use it in the first place!  Select the rest, and click *join* beams.  Because the rest is the only thing getting in the way of the beam already being joined, so it's the only thing that needs to have a property set, and that property is indeed *join*, not *break*.  It's just kind of a weird special special case that the "break inner beams" really means "*and* join the *outer* beams if they weren't already".

**Comment by Skip12ish** [comment]
Yes Sir!Selecting the "rest" first, did the trick! 
I sure thought I tried that yesterday, but obviously I didn't. 

Thanks for the clarifications! Glad its not a bug or quirk.

Thanks again!

**Comment by davidstephengrant** [comment]
Closing this as by design.
